Connectors/ArcGIS/Speckle.Connectors.ArcGIS3/HostApp/ArcGISColorManager.cs

```diff
@@ -316,6 +316,10 @@ private void ProcessFeatureLayerColors(FeatureLayer layer, int displayPriority)
     int count = 1;
     using (RowCursor rowCursor = layer.Search())
     {
+      if (rowCursor is null)
+      {
+        return;
+      }
```

Comment:
Does this mean that we can skip an item in a layer if it's null?

Could this happen for a valid layer with valid fields?

Maybe I'm understanding this wrong 😅 but if this is the case we should at least log it somehow.

Reply:
added a comment 🙌